test/unit/BPool/BPool_ExitswapPoolAmountIn.t.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import {BPoolBase} from './BPoolBase.sol';
+
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+import {BMath} from 'contracts/BMath.sol';
+import {BNum} from 'contracts/BNum.sol';
+import {IBPool} from 'interfaces/IBPool.sol';
+
+contract BPoolExitSwapPoolAmountIn is BPoolBase, BMath {
+  // Valid scenario:
+  address public tokenOut;
+  uint256 public tokenOutWeight = 3e18;
+  uint256 public tokenOutBalance = 400e18;
+  uint256 public totalWeight = 9e18;
+  uint256 public poolAmountIn = 1e18;
+  // calcSingleOutGivenPoolIn(400, 3, 100, 9, 1, 10^(-6))
+  uint256 public expectedAmountOut = 11.880392079733333329e18;
+
+  function setUp() public virtual override {
+    super.setUp();
+    tokenOut = tokens[1];
+    bPool.set__records(tokenOut, IBPool.Record({bound: true, index: 0, denorm: tokenOutWeight}));
```

Comment:
...we don't use it for anything, but index is wrong

Reply:
is because is not being used, set to default..